transformers/storage/bid_storage.go

```diff
@@ -26,6 +26,7 @@ var (
 	Tau    = "tau"
 	Kicks  = "kicks"
 	Live   = "live"
+	Vow    = "vow"
```

Comment:
totally doesn't have to be part of this PR, but I wonder if we should rename this file? 🤔

Reply:
Yeah good point. Seems like these vars might be more at home in `constants/keys.go`, with the rest of our key/column strings